client/src/components/PitchDoc/ClaimPitchModal.tsx

```diff
@@ -0,0 +1,295 @@
+    const elementIdx = selectedTeams.indexOf(team);
+    if (elementIdx === notFoundIdx) {
+      const addedElements = selectedTeams.concat(team);
+
+      setSelectedTeams(addedElements);
+    } else {
+      const removedElements = selectedTeams.filter(
+        (element) => element !== team,
+      );
+
+      setSelectedTeams(removedElements);
+    }
+  };
+
+  const getTeams = useCallback(async (): Promise<void> => {
+    const resp = await getOpenTeams(pitch._id);
+
+    if (!isError(resp) && resp.data) {
+      setOpenTeams(resp.data.result);
+    }
+  }, [pitch._id]);
+
+  const getUser = useCallback(async () => {
+    const contributors: { [name: string]: string } = {};
+    for (const userId of pitch.assignmentContributors) {
+      const res = await loadUser(userId);
+      if (!isError(res)) {
+        contributors[
+          `${res.data.result.firstName} ${res.data.result.lastName}`
+        ] = temp_pfp;
+      }
+    }
+    setPitchContributors(contributors);
+    if (pitch.pitchAuthor) {
+      const res = await loadUser(pitch.pitchAuthor);
+      if (!isError(res)) {
+        setPitchAuthor(
+          `${res.data.result.firstName} ${res.data.result.lastName}`,
+        );
+      }
+    }
+    if (pitch.approvedBy) {
+      const res = await loadUser(pitch.approvedBy);
+      if (!isError(res)) {
+        setApprovedBy(
+          `${res.data.result.firstName} ${res.data.result.lastName}`,
+        );
+      }
+    }
+  }, [
+    pitch.assignmentContributors,
+    pitch.pitchAuthor,
+    pitch.approvedBy,
+    temp_pfp,
+  ]);
+
+  useEffect(() => {
+    getTeams();
+    setSelectedTeams([]);
+    getUser();
+    setPitchLink('');
+  }, [firstOpen, getTeams, getUser]);
+
+  return (
+    <>
+      <Modal
+        className="claim-pitch-modal"
+        onClose={() => {
+          setFirstOpen(false);
+        }}
+        onOpen={() => {
+          setFirstOpen(true);
+        }}
+        open={firstOpen}
+        trigger={<PitchCard pitch={pitch} openTeams={openTeams}></PitchCard>}
+        closeIcon
+      >
+        <Modal.Actions>
+          <div className="claim-pitches-wrapper">
+            <div className="pitch-info">
+              <div className="topics-wrapper">
+                <Label.Group style={{ marginBottom: 10 }} circular>
+                  {pitch.topics.map((topic, idx) => (
+                    <>
+                      <Label
+                        style={{
+                          backgroundColor:
+                            interestsButtons[enumToInterestButtons[topic]],
+                        }}
+                        key={idx}
+                      >
+                        {enumToInterestButtons[topic]}
+                      </Label>
+                    </>
+                  ))}
+                </Label.Group>
+              </div>
+              <div className="pitch-name"> {pitch.name} </div>
+              <span className="submitted-reviewed-by">
+                {' '}
```

Comment:
Is there a reason for this? I feel like the strings should be built in a better way if we have to JSX string fragments throughout the JSX

Reply:
Prefer the `String: ${variable}` method